cli/enrol.yaml

```diff
@@ -0,0 +1,72 @@
+## Every recipe file must provide at least the component property.
+## Based on the plugin type plugin specific files files will be generated.
+component: enrol_plugintest
+
+## Human readable name of the plugin.
+name: Enrol plugin
+
+## Human readable release number.
+release: "0.1.0"
+
+## Plugin version number, e.g. 2016062100. Will be set to current date if left empty.
+#version: 2016121200
+
+## Required Moodle version, e.g. 2015051100 or "2.9".
+requires: "2.9"
+
+## Plugin maturity level. Possible options are MATURIY_ALPHA, MATURITY_BETA,
+## MATURITY_RC or MATURIY_STABLE.
+maturity: MATURITY_BETA
+
+## Copyright holder(s) of the generated files and classes.
+copyright: 2016 Alexandru Elisei <alexandru.elisei@gmail.com>, David Mudrák <david@moodle.com>
+
+## Features flags can control generation of optional files/code fragments.
+features:
+    readme: true
+    license: true
+    install: true
+    settings: true
+
+## Allows manual enrolment. Requires enrol/<enrolname>:enrol capability.
+allow_enrol: 'false'
```

Comment:
Naming things is hard in programming. I generally understand "feature" here as "a distinctive attribute or aspect of something" with synonyms like "characteristic", "attribute" or "property" (https://www.google.com/#q=define%3A%20features).

From this perspective, it makes perfect sense to me and I would prefer having `features:` and `<plugintype>_features:` in our recipe files.